scripts/sql/postgre/apolloconfigdb.sql

```diff
@@ -0,0 +1,442 @@
+--
+-- Copyright 2023 Apollo Authors
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+-- http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+\c "ApolloConfigDB"
+/* # Dump of table app */
+/* # ------------------------------------------------------------ */
+
+DROP TABLE IF EXISTS apolloconfig."App";
+
+CREATE TABLE apolloconfig."App"
+(
+    "Id"                        BIGSERIAL,                                       -- COMMENT '主键'
+    "AppId"                     varchar(64)  NOT NULL DEFAULT 'default',         -- COMMENT 'AppID'
+    "Name"                      varchar(500) NOT NULL DEFAULT 'default',         -- COMMENT '应用名'
+    "OrgId"                     varchar(32)  NOT NULL DEFAULT 'default',         -- COMMENT '部门Id'
+    "OrgName"                   varchar(64)  NOT NULL DEFAULT 'default',         -- COMMENT '部门名字'
+    "OwnerName"                 varchar(500) NOT NULL DEFAULT 'default',         -- COMMENT 'ownerName'
+    "OwnerEmail"                varchar(500) NOT NULL DEFAULT 'default',         -- COMMENT 'ownerEmail'
+    "IsDeleted"                 bit          NOT NULL DEFAULT b'0',              -- COMMENT '1: deleted, 0: normal'
```

Comment:
Looks like this is wrong.
```suggestion
    "IsDeleted"                 BOOL        NOT NULL DEFAULT false,             -- COMMENT '1: deleted, 0: normal'
```

Reply:
fixed, thanks